src/sentry/testutils/cases.py

```diff
@@ -685,3 +689,78 @@ def setUp(self):
 
     def assertDialogSuccess(self, resp):
         assert 'window.opener.postMessage(' in resp.content
+
+
+class SnubaTestCase(TestCase):
+    def setUp(self):
+        super(SnubaTestCase, self).setUp()
+
+        assert requests.post(snuba.SNUBA + '/tests/drop').status_code == 200
+
+    def __wrap_event(self, event, data, primary_hash):
+        # TODO: Abstract and combine this with the stream code in
+        #       getsentry once it is merged, so that we don't alter one
+        #       without updating the other.
+        return {
+            'group_id': event.group_id,
+            'event_id': event.event_id,
+            'project_id': event.project_id,
+            'message': event.message,
+            'platform': event.platform,
+            'datetime': event.datetime,
+            'data': data,
+            'primary_hash': primary_hash,
+        }
+
+    def create_event(self, *args, **kwargs):
```

Comment:
Not for this patch, but I feel like we half-implement the event manager a lot during tests, I'd like to find a better way to do this that's more sustainable sometime.